app/scss/_base.scss

```diff
@@ -36,6 +36,7 @@ body {
 
 h1 {
   font-family: $font-accent;
+  margin: $md;
```

Comment:
Are you sure you want right and left margin too? That will affect how h1's line up with p's, etc.

Reply:
Good catch! Fixed.